packages/eas-cli/README.md

```diff
@@ -19,6 +19,28 @@ npm install -g eas-cli
 yarn global add eas-cli
 ```
 
+## Enforcing eas-cli version for your project
+
+If you want to enforce the **eas-cli** version for your project, use the `"cli.version"` field in eas.json. Installing **eas-cli** as your project dependency is strongly discouraged.
```

Comment:
```suggestion
If you want to enforce the `eas-cli` version for your project, use the `"cli.version"` field in **eas.json**. Installing `eas-cli` to your project dependencies is strongly discouraged because it may cause dependency conflicts that are difficult to debug.
```